flows/exam-1.yml

```diff
@@ -28,24 +28,28 @@ rules:
         permissions: []
 
     -
-        if_in_facility: test_center
-        if_session_duration_shorter_than_minutes: 0.5
-        permissions: [view, submit_answer, end_session, see_session_time, lock_down_as_exam_session]
+        if_has_role: [instructor]
+        permissions: [view, submit_answer, change_answer, end_session, see_correctness, see_session_time]
```

Comment:
`change_answer` and `see_correctness` is not a good combo if using multiple-choice. I think it's preferable to add `change_answer` using per-page permissions.